src/App.tsx

```diff
@@ -0,0 +1,139 @@
+import { useState, FormEvent } from 'react'
+import logoImg from './assets/logo.jpg'
+import carro from './assets/car.gif'
+import './App.css'
+
+interface InfoProps {
+  combustivel: number;
+  pedagio: number;
+  total: number;
+};
+
+function App() {
+  const [distancia, setDistancia] = useState(0);
```

Comment:
Estamos usando typescript, voce pode tipar os useState para evitar que ele tenha algum tipo diferente.

Voce pode fazer isso para todos os `useState`.

```suggestion
  const [distancia, setDistancia] = useState<number>(0)
```

Reply:
Feito para todos do useState